.circleci/config.yml

```diff
@@ -0,0 +1,160 @@
+defaults: &defaults
+  docker:
+    - image: circleci/node:10-browsers
+  working_directory: ~/origin
+
+db: &db
+  docker:
+    - image: circleci/node:10-browsers
+    - image: circleci/postgres:9.6-alpine-ram
+      environment:
+        POSTGRES_USER: postgres
+        POSTGRES_DB: travis_ci_test
```

Comment:
super nit: would that make sense to call that db circle_ci_test ?

Reply:
Updated 🙂